docs/sql-reference/sql-functions/Window_function.md

```diff
@@ -196,6 +198,44 @@ from int_t where property in ('odd','even');
 +----+----------+------------------+
 ~~~
 
+### CUME_DIST()
+
+The CUME_DIST() function calculates the cumulative distribution of a value within a partition, indicating its relative position as a percentage of values less than or equal to the value in the current row. With a range of 0 to 1, it's useful for percentile calculations and data distribution analysis.
+
```

Comment:
please add some notes for the case without order by.
```
**This function should be used with ORDER BY to sort partition rows into the desired order. Without ORDER BY, all rows are peers and have value N/N = 1, where N is the partition size.**
```

Reply:
Done.